common/djangoapps/track/middleware.py

```diff
@@ -70,7 +83,12 @@ def process_request(self, request):
 
             views.server_track(request, request.META['PATH_INFO'], event)
         except:
-            pass
+            ## Why do we have the overly broad except?
```

Comment:
As an aside: An instrumentation problem *will* fundamentally cause user-facing errors if we do anything user-facing with the data.